src/nunit.analyzers.tests/Targets/TestCaseUsage/TestCaseUsageAnalyzerTestsAnalyzeWhenReturnTypeIsAsyncVoid.cs

```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace NUnit.Analyzers.Tests.Targets.TestCaseUsage
+{
+    class TestCaseUsageAnalyzerTestsAnalyzeWhenReturnTypeIsAsyncVoid
+    {
+        [Test]
```

Comment:
note, this isn't a `TestCase`, but a `Test`

Reply:
Perhaps we will - at some time - need a test for both cases. But for now I think it is fine